### DISCUSSION TITLE
How disable /page/number in sitemap.xml?

### DISCUSSION THREAD

**Discussion by silbox-ch** [GetPublii/Publii#1799] [Help]
Hello, I've noticed this in sitemap.xml. How do we disable this?

![image](https://github.com/user-attachments/assets/f34c503c-dc71-49ec-bf0a-4e1d0a5961ca)

**Comment by bobmitro** [comment]
If you want search engine bots to index paginated pages on your website, make sure to include them in your sitemap. If not, go to the SEO options in Site Settings and disable indexing for homepage, tag, or author pagination.

Example: 
<img width="961" alt="screenshot 2024-11-08 at 20 57 45" src="https://github.com/user-attachments/assets/ec568e0e-e09e-4fb6-b7c1-df5912aeb314">
